src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -0,0 +1,1240 @@
+
+    # TODO
+    # Step 2: Create FSS database which includes at least one FSS site near
+    # CBSD location 'X'.
+
+    # TODO
+    # Step 3: Create GWBL database which includes at least one GWBL site near
+    # CBSD location 'X'.
+
+    # Step 4: Trigger daily activities
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    # Step 5: Send Grant Request 'G1'.
+    # Forming of 'G1' grants for all the configured grants
+    addCbsdIdsToRequests(cbsd_ids, config['grantRequests'][0])
+    grant_request_g1 = {'grantRequest': config['grantRequests'][0]}
+
+    grant_response_g1 = self._sas.Grant(grant_request_g1)['grantResponse']
+
+    # Check the grant response code is as expected (SUCCESS or INTERFERENCE)
+    grant_ids = []
+    for resp in grant_response_g1:
+      self.assertIn(resp['response']['responseCode'], config['expectedResponseCodes'][0])
+      grant_ids.append(resp['grantId'])
+
+    # Step 6: Send relinquishment request for grant 'G1'
+    relq_requests = {'relinquishmentRequest': []}
+    for cbsd_id, grant_id in zip(cbsd_ids, grant_ids):
+      # Update Relinquishment Request content
+      relq_requests['relinquishmentRequest'].append({
+        'cbsdId': cbsd_id,
+        'grantId': grant_id
+      })
+    relq_responses = self._sas.Relinquishment(relq_requests)['relinquishmentResponse']
+    self.assertEqual(len(relq_responses), len(relq_requests['relinquishmentRequest']))
+
+    # Check relinquishment response
+    for relq_resp in relq_responses:
+      self.assertEqual(relq_resp['response']['responseCode'], 0)
+
+    del relq_requests, relq_responses
+    del grant_request_g1, grant_response_g1
+
+    # TOD0
+    # Step 7: Modify the GWBL database to include a modified version of the GWBL 'W'.
+    # 'W' is moved further than 150 kms from FSS site.
+
+    # Step 8: Trigger daily activities
+    self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
+
+    # Step 9: Send Grant Request 'G2'
+    # Forming of 'G2' grants for all the configured grants
+    addCbsdIdsToRequests(cbsd_ids,config['grantRequests'][1])
+    grant_request_g2 = {'grantRequest': config['grantRequests'][1]}
+
+    grant_response_g2 = self._sas.Grant(grant_request_g2)['grantResponse']
+
+    # Check the grant response code is as expected (SUCCESS or INTERFERENCE).
+    for resp in grant_response_g2:
+      self.assertIn(resp['response']['responseCode'], config['expectedResponseCodes'][1])
+
+    del grant_request_g2, grant_response_g2
+
+  def generate_FDB_7_default_config(self, filename):
+    """Generates the WinnForum configuration for FDB.7"""
+
+    # Load devices info
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Update the FCC ID of CBSDs with the FCC ID 'F_ID' from FCC ID database.
+    device_a['fccId'] = "F_ID"
```

Comment:
`F_ID` in the spec was meant as a var name, not a specific value. Please use different values for these two.

Reply:
FDB.7        changed to "F_ID_a" and "F_ID_b".